### PR TITLE
UI matching

### PR DESCRIPTION
**Pull request by mtlssica**
css for matching page. 
User with match tested -> username: melo@sad, password: melosad
User without match tested -> username: test@test.com, password: password

### CODE REVIEW

**Comment by ca8marti-school** [comment]
Users may have been matched with melo 1-3 because of testing done with integrating the matching algorithm. Most of the responses are 0 (first response), which is what was used for most of the tests

File: src/Matches.js [hunk 1]
```diff
@@ -82,42 +82,49 @@ class Matches extends React.Component {
...
+
             }
-            matchingSection = 
-                <div>
-                    <td id='Matches-congrats'>
-                        Congratulations, these people want to be your friend!
-                    </td>
+            matchingSection = <h3 id='Matches-congrats'>
+                <p>Congratulations,</p>
+                <p>You got potential pal-ed!</p>
```
Comment (ca8marti-school) [inline src/Matches.js]: Not sure what pal-ed means, maybe change it to congratulations you have a match and based on the number of matches have a different response?

File: src/Matches.js [hunk 2]
```diff
@@ -82,42 +82,49 @@ class Matches extends React.Component {
...
                     {userItems}
-                </div>
+                </p>
+                </h3>
         }
         else if (this.state.matchesExist == "not exists") {
             matchingSection = <h2 id='Matches-none'>
-                Sorry, No one wants to be your friend for now. Please try again later :(
+                <p>Sorry, no one met the matching criteria you set.</p> 
+                <p>We suggest you to edit your profile.</p>
```
Comment (echanlee) [inline src/Matches.js]: We suggest that you edit your profile or wait for more users

File: src/Matches.js [hunk 3]
```diff
@@ -82,42 +82,49 @@ class Matches extends React.Component {
...
         }
         return(
             <div id='Matches-section'>
-                <h2 id='Matches-header'>Matches</h2>
-                <div>{this.state.name}</div>
-                <Link to={{pathname: '/main', state: {id: this.state.userId}}}>Back to Swiping!</Link>
                 <div id='Matches-section'>
                     {matchingSection}
+                </div> 
+                <button id = 'messagingButton'>Say Hello</button>
```
Comment (echanlee) [inline src/Matches.js]: did you test this to make sure it works? Also does this specify what user you are trying to say hello to

**Review by echanlee — CHANGES_REQUESTED** [review]

**Pull request merged**